### PR TITLE
Added my blog post information and screen shot to student.yml

### PR DESCRIPTION
**Pull request by rhemon**
Presently I just added to student.yml file since there wasn’t any other option for blog post. Once I am done with my creating blog task, I plan on adding a blog section to the website.

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -92,3 +92,8 @@
   facebook: https://facebook.com/cuteaditya1
   email: blusinghaditya@gmail.com
   bio: Programmer & Web Designer, Second year GCI!
+
+- name: Ridhwanul Haque
+  github: rhemon
+  blog_post: https://rhemon.github.io/blog/gci/2016/11/30/00-fossassia-me-what-why/
+  blog_img: rhemon-blog.png
```
Comment (robbyoconnor) [inline _data/students.yml:99]: You should link to yout blog, not a single post!

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

**Comment by rhemon** [comment]
Done, fixed it

**Comment by rhemon** [comment]
Do i have to do a new pull request?

**Comment by robbyoconnor** [comment]
Nope -- if you pushed the same branch as this PR -- you're solid. :+1:  -- I can't merge this.

**Comment by robbyoconnor** [comment]
I need to wait on this to be merged

**Review by daminisatya — APPROVED** [review]

**Comment by daminisatya** [comment]
Merged it! 👍

**Pull request merged**